wxt_cidrbot/cidrbot.py

```diff
@@ -166,17 +169,49 @@ def webhook_request(self, event):
             if org_id == self.orgID:
                 self.message_event(json_string, event_type, webex_msg_sender)
 
+    def check_message_overflow(self, message, room_id, message_id, pt_id, request_type):
+        split_keyword = 'Repo:'
+
+        if len(message) < 4800:
+            return False
+
+        if split_keyword in message:
+            split_message = message.split(split_keyword)
+            message_first_part = ""
+            remainder_message = ""
+            for repo in split_message:
+                self.logging.debug("Length of repo %s", len(repo))
+                if len(message_first_part) + len(repo) < 4800:
+                    if '**All Issues:**' not in repo:
+                        message_first_part += split_keyword
+                    message_first_part += repo
+                elif len(remainder_message) + len(repo) < 4800:
+                    remainder_message += "Repo:" + repo
+                else:
+                    remainder_message = "One repo has too many issues and exceeds the webex message limit."
+
+        if request_type == 'daily_message':
+            self.send_wbx_msg(room_id, message_first_part, pt_id)
```

Comment:
does it make sense to simply this block and pull the send_wbx_msg outside of the ifs?

Reply:
Moved self.send_wbx_msg() outside of block, unnecessary duplication of line, since it runs in both cases